packages/inter-protocol/src/vaultFactory/vaultManager.js

```diff
@@ -68,13 +81,16 @@ const trace = makeTracer('VM', false);
  * @property {Amount<'nat'>}  totalCollateral    present sum of collateral across all vaults
  * @property {Amount<'nat'>}  totalDebt          present sum of debt across all vaults
  * @property {Amount<'nat'>}  retainedCollateral collateral held as a result of not returning excess refunds
- *                                                from AMM to owners of vaults liquidated with shortfalls
+ *                                                to owners of vaults liquidated with shortfalls
+ * @property {Amount<'nat'>}  liquidatingCollateral  present sum of collateral in vaults sent for liquidation
+ * @property {Amount<'nat'>}  liquidatingDebt        present sum of debt in vaults sent for liquidation
  *
  * @property {Amount<'nat'>}  totalCollateralSold       running sum of collateral sold in liquidation
  * @property {Amount<'nat'>}  totalOverageReceived      running sum of overages, central received greater than debt
- * @property {Amount<'nat'>}  totalProceedsReceived     running sum of central received from liquidation
- * @property {Amount<'nat'>}  totalShortfallReceived    running sum of shortfalls, central received less than debt
- * @property {number}         numLiquidationsCompleted  running count of liquidations
+ * @property {Amount<'nat'>}  totalProceedsReceived     running sum of minted received from liquidation
+ * @property {Amount<'nat'>}  totalShortfallReceived    running sum of shortfalls, minted received less than debt
+ * @property {number}         numLiquidationsCompleted  running count of liquidated vaults
```

Comment:
Let's be clear one what "liquidated" means. The term "revertLiquidation" implies that it _was liquidated_ and that's being reverted. Is that right? Then it's reasonable to infer that a "liquidation completed" could be "reverted"?

WDYT of "liquidation aborted"? Then "completed" unambiguously means that it reached the end without an abort.